nextcloud-aio-helm-chart/templates/nextcloud-aio-collabora-deployment.yaml

```diff
@@ -21,7 +21,8 @@ spec:
         io.kompose.service: nextcloud-aio-collabora
     spec:
       containers:
-        - env:
+        - args: {{ .Values.ADDITIONAL_COLLABORA_OPTIONS | default [] }}
```

Comment:
```suggestion
        - args: {{ .Values.ADDITIONAL_COLLABORA_OPTIONS | default list | toJson }}
```

Reply:
Thanks Richard! Should we maybe also convert the whole `ADDITIONAL_COLLABORA_OPTIONS ` to a list and to json? Or does it work like so, e.g. if it is set to `ADDITIONAL_COLLABORA_OPTIONS: ['--o:security.seccomp=true'] `?

Reply:
Thanks Richard! :)

Reply:
Should be adjusted now. Please re-review :)